### PR TITLE
Implemented RESX file comparison in integration test

### PR DESCRIPTION
**Pull request by Benglin**
Background
-----
Every time a new localizable string is needed, it is added to `Resources.resx` file. Subsequently, the `Resources.en-US.resx` file should be updated to match `Resources.resx` file exactly, so the practice is usually to overwrite `Resources.en-US.resx` with `Resources.resx` file.

This new test case `AllResxFilesShouldMatchEnUsCounterparts` is added to scan `Dynamo/src` directory and sub-directories for `Resources.resx` files, and then compare them with their `Resources.en-US.resx` counterparts, ensuring they are always the same.

To make this test pass, I have added two new `Resources.en-US.resx` files that were missing (but did not cause any issue):

1. `src/Legacy/GraphToDSCompiler/Properties/Resources.en-US.resx`
2. `src/Tools/InstallUpdate/Resources.en-US.resx`

Hi @Sancerio, please take a look. Thanks!

### CODE REVIEW

File: test/System/IntegrationTests/IntegrityTests.cs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        /// files, and tries to compare them with their "Resources.en-US.resx" 
+        /// counterparts to make sure they are exactly the same. The guideline 
+        /// is, everytime "Resources.resx" is updated, it should be copied to 
+        /// overwrite "Resources.en-US.resx" so that they are always in-sync.
+        /// </summary>
+        /// 
+        [Test]
+        public void AllResxFilesShouldMatchEnUsCounterparts()
+        {
+            var executingAsmPath = Assembly.GetExecutingAssembly().Location;
```
Comment (sharadkjaiswal) [inline test/System/IntegrationTests/IntegrityTests.cs]: This test may not work on CI system, sometimes we just copy binaries to run the tests. On the test machine, we may not have the source code to run this test.

Comment (Benglin) [inline test/System/IntegrationTests/IntegrityTests.cs]: I see, do we do that for all the time or under specific build configuration? If we do this all the time, then I may have to make the test succeed when `Dynamo/src` folder cannot be found.

**Comment by Sancerio** [comment]
@Benglin, besides the CI issue (whether it would run on CI or not), LGTM.

**Comment by sharadkjaiswal** [comment]
We can have some build integrity tests, which can run after the build on the CI system.

**Comment by pboyer** [comment]
@Benglin Closing due to lack of activity.

**Pull request closed without merging**